eos-updater-flatpak-installer/docs/eos-updater-flatpak-installer.8

```diff
@@ -0,0 +1,100 @@
+.\" Manpage for eos-updater-flatpak-installer.
+.\" Documentation is under the same licence as the eos-updater package.
+.TH man 8 "13 Apr 2017" "1.0" "eos\-updater\-flatpak\-installer man page"
+.\"
+.SH NAME
+.IX Header "NAME"
+eos\-updater\-flatpak\-installer — Endless OS Updater - Flatpak Installer
+.\"
+.SH SYNOPSIS
+.IX Header "SYNOPSIS"
+.\"
+\fBeos\-updater\-flatpak\-installer [\-m \-\-mode\fB] [\-p \-\-pull\fB]
```

Comment:
You can drop the `\fB`s from before `]` — they’re only needed if an argument takes a parameter, and in that case are paired with an `\fP` before the parameter.